controller/cmd/exporter-set-controller/main.go

```
@@ -159,9 +160,11 @@ func main() {
 // Add new provisioners here as they are implemented.
 func selectProvisioner(name string) (exporterset.Provisioner, error) {
 	switch name {
+	case cuttlefish.ProvisionerName:
+		return cuttlefish.New(version), nil
 	case qemu.ProvisionerName:
 		return qemu.New(version), nil
 	default:
-		return nil, fmt.Errorf("unknown provisioner %q; supported: %s", name, qemu.ProvisionerName)
+		return nil, fmt.Errorf("unknown provisioner %q; supported: %s, %s", name, qemu.ProvisionerName, cuttlefish.ProvisionerName)
```

golangci-lint (2.13.2)

[error] The line is 125 characters long, which exceeds the maximum of 120 characters.

(lll)